### PR TITLE
dx-1205 changed the sequence of the overall main menu

### PR DESCRIPTION
**Pull request by marinafrydenb**

### CODE REVIEW

**Review by asbjornu — APPROVED** [review]
If this works, then okay, but the field should really have been named **`menu_order`**, not `menu-order`. I think we should change this before we go live with the new design.

**Review by arebra — COMMENTED** [review]
Where is the menu_order change?

**Comment by asbjornu** [comment]
> Where is the menu_order change?

@arebra, in #679. :)

**Review by arebra — APPROVED** [review]

File: Gemfile [hunk 1]
```diff
@@ -15,4 +15,4 @@ gem 'html-proofer'
 gem 'html-proofer-unrendered-markdown'
 gem 'jekyll-redirect-from'
 gem "kramdown", ">= 2.3.0"
-gem 'swedbank-pay-design-guide-jekyll-theme', '~> 1.7'
+gem 'swedbank-pay-design-guide-jekyll-theme', '~> 1.8'
```
Comment (asbjornu) [inline Gemfile:18]: 1.8.1 brings in the `menu_order` renaming.

```suggestion
gem 'swedbank-pay-design-guide-jekyll-theme', '~> 1.8.1'
```

**Review by asbjornu — CHANGES_REQUESTED** [review]

**Comment by asbjornu** [comment]
Superseded by #679.

**Pull request closed without merging**